syncd/scripts/brcm_common_config_ut.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash 
```

Comment:
@geans-pin could you please give some guideline how to run this UT? 
I suppose UT is triggered in PR and it can cover all hwskus for all broadcom platforms.
For every hwsku, there should be a sample file, after merging common config.bcm file in config_syncd_bcm, the new generated config.bcm should be same with the sample file.
Any other thoughts here?

Reply:
No,we don't need to have sample to compare. Doing this you will need to maintain extra sample file, and when you update common or platform config, this file has to be modified as well.

Note, the config merged is happened in the runtime not static merged in the build time. For one platform, it will only generate one merged config in the runtime. The UT scripted has to be run in the runtime after merged in the syncd docker. It will check if common config overwrite and non overwrite properties were merged successfully to target config.


Geans